account_invoice_supplier_self_invoice/README.rst

```diff
@@ -0,0 +1,66 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+   :target: https://www.gnu.org/licenses/agpl-3.0-standalone.html
+   :alt: License: AGPL-3
+
+=====================================
+Account invoice purchase self invoice
+=====================================
+
+This module allows the configuration of the suppliers in order to define self
+invoices.
+Self invoices are created by the customer in the name of the provider.
+The provider must allow the customer to create them.
+
+Usage
+=====
+
+#. Go to 'Accounting/Invoicing > Purchases > Master Data > Vendors'
+#. Modify a Provider and click 'Set Self Invoice' inside the page 'Sales &
+   Purchases'
+#. Go to 'Accounting/Invoicing > Purchases > Documents > Vendors Bills'
+#. Create an invoice for the provider and validate it
+#. The self invoice is accessible through the button: Self invoice
+#. You can create an invoice for the provider without the self invoice if you
+   uncheck 'Set self invoice'
+
+.. image:: https://odoo-community.org/website/image/ir.attachment/5784_f2813bd/datas
+   :alt: Try me on Runbot
+   :target: https://runbot.odoo-community.org/runbot/159/11.0
+
+
+Bug Tracker
+===========
+
+Bugs are tracked on `GitHub Issues
+<https://github.com/OCA/account-invoicing/issues>`_. In case of trouble, please
+check there if your issue has already been reported. If you spotted it first,
+help us smash it by providing detailed and welcomed feedback.
+
+Credits
+=======
+
+Images
+------
+
+* Odoo Community Association: `Icon <https://github.com/OCA/maintainer-tools/blob/master/template/module/static/description/icon.svg>`_.
+
+Contributors
+------------
+
+* Enric Tobella <etobella@creublanca.es>
+* Nikul Chaudhary <nikulchaudhary2112@gmail.com>
```

Comment:
This kind of migration is not enough to add yourself as contributor.

Reply:
@etobella It's Migrated module and tested all file for that, If I changed any singal file keep added your contributor.

Reply:
I think the contribution is enough for being here, but not on authors.

Reply:
> I think the contribution is enough for being here, but not on authors.

Yes @pedrobaeza , I added in only contribution section.